ompi/mpi/java/java/Comm.java

```diff
@@ -1170,7 +1170,7 @@ public final void abort(int errorcode) throws MPIException
 
 	/**
 	 * Associates a new error handler with communicator at the calling process.
-	 * <p>Java binding of the MPI operation {@code MPI_ERRHANDLER_SET}.
+	 * <p>Java binding of the MPI operation {@code MPI_COMM_ERRHANDLER_SET}.
```

Comment:
This should be `MPI_COMM_SET_ERRHANDLER`.

Reply:
I fixed it.